mhkit/wave/io/hindcast/wind_toolkit.py

```diff
@@ -478,7 +483,11 @@ def request_wtk_point_data(
             meta = meta.reset_index(drop=True)
 
         # Save the retrieved data and metadata to cache.
-        handle_caching(hash_params, cache_dir, data=data, metadata=meta)
+        handle_caching(
+            hash_params,
+            cache_dir,
+            cache_content={"data": data, "metadata": meta, "write_json": None},
+        )
```

Comment:
handle_caching reduced to only 5 inputs
https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#max-args